Pylint errors :(. @jmcarp -> pre-commits are helpful :). I save a lot of time thanks to them :)


```
tests/providers/amazon/aws/utils/test_emailer.py:20:0: W0611: Unused import copy (unused-import)
tests/providers/amazon/aws/utils/test_emailer.py:21:0: W0611: Unused import os (unused-import)
tests/providers/amazon/aws/utils/test_emailer.py:22:0: W0611: Unused import tempfile (unused-import)
tests/providers/amazon/aws/utils/test_emailer.py:23:0: W0611: Unused import unittest (unused-import)
```

--- a/airflow/providers/amazon/aws/utils/__init__.py
+++ b/airflow/providers/amazon/aws/utils/__init__.py
@@ -1,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.

--- a/airflow/providers/amazon/aws/utils/emailer.py
+++ b/airflow/providers/amazon/aws/utils/emailer.py
@@ -1,0 +1,49 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Airflow module for email backend using AWS SES"""
+
+from typing import List, Optional, Union
+
+from airflow.providers.amazon.aws.hooks.ses import SESHook
+
+
+def send_email(
+    to: Union[List[str], str],
+    subject: str,
+    html_content: str,
+    files: Optional[List] = None,
+    cc: Optional[Union[List[str], str]] = None,
+    bcc: Optional[Union[List[str], str]] = None,
+    mime_subtype: str = 'mixed',
+    mime_charset: str = 'utf-8',
+    conn_id: Optional[str] = None,
+    **kwargs,
+) -> None:
+    """Email backend for SES."""
+    hook = SESHook(aws_conn_id=conn_id)
+    hook.send_email(
+        mail_from=None,
+        to=to,
+        subject=subject,
+        html_content=html_content,
+        files=files,
+        cc=cc,
+        bcc=bcc,
+        mime_subtype=mime_subtype,
+        mime_charset=mime_charset,
+    )

--- a/tests/providers/amazon/aws/utils/__init__.py
+++ b/tests/providers/amazon/aws/utils/__init__.py
@@ -1,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.

--- a/tests/providers/amazon/aws/utils/test_emailer.py
+++ b/tests/providers/amazon/aws/utils/test_emailer.py
@@ -1,0 +1,42 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from unittest import mock
+
+from airflow.providers.amazon.aws.utils.emailer import send_email
+
+
+@mock.patch("airflow.providers.amazon.aws.utils.emailer.SESHook")
+def test_send_email(mock_hook):
+    send_email(
+        to="to@test.com",
+        subject="subject",
+        html_content="content",
+    )
+    mock_hook.return_value.send_email.assert_called_once_with(
+        mail_from=None,
+        to="to@test.com",
+        subject="subject",
+        html_content="content",
+        bcc=None,
+        cc=None,
+        files=None,
+        mime_charset="utf-8",
+        mime_subtype="mixed",
+    )

